docs/docusaurus.config.js

```diff
@@ -87,6 +121,12 @@ const config = {
             label: "GitHub",
             position: "right",
           },
+          {
```

Comment:
I think you need to manually configure the docSidebar for the API. The navigation for the API includes the learn and build sections (which it shouldn't)

![image](https://github.com/bob-collective/bob/assets/14966470/de0caa72-5081-49a0-b9c9-c78be9e90039)